CHANGELOG.md

```diff
@@ -46,6 +46,19 @@ Swift Next
     // OK, <U where U: Equatable> has broader visibility than <T where T == Int>
     override func foo() where U: Equatable { ... } 
   }
+
+* [SR-75][]:
+
+  Unapplied references to protocol methods methods are now supported. Previously this
+  only worked for methods defined in structs, enums and classes.
+
+  ```swift
+  protocol Cat {
+    func play(catToy: Toy)
+  }
+
+  let fn = Cat.play
```

Comment:
[Require parameter names when referencing to functions](https://forums.swift.org/t/require-parameter-names-when-referencing-to-functions/27048)?

```diff
-let fn = Cat.play
+let fn = Cat.play(catToy:)
```

Reply:
Not actually required, but I agree it makes the example more clear.